### ISSUE TITLE
Is there a keras version?

### ISSUE THREAD

**Issue by AbnerAI** [utkuozbulak/adaptive-segmentation-mask-attack#1]
Hi, I want to use ASMA to attack the model saved by keras, but I encounter some difficulties when customizing the loss. Is there a keras version of ASMA?

**Comment by utkuozbulak** [comment]
Hello,

No, we only worked with Pytorch. I'm not sure if someone else implemented it in Keras.
